code/index.html

```diff
@@ -4,18 +4,46 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Project GitHub Tracker</title>
+
+  <!-- Chart 👇 -->
+  <script src="https://cdn.jsdelivr.net/npm/chart.js"></script>
+
   <link rel="stylesheet" href="./style.css" />
+
+  <!-- Google Fonts 👇 -->
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Shippori+Antique+B1&family=Shizuru&display=swap" rel="stylesheet">
+
+  <title>GitHub Tracker by Camilla</title>
 </head>
+
 <body>
-  <h1>GitHub Tracker</h1>
-  <h2>Projects:</h2>
-  <main id="projects"></main>
+  <header>
+    <div class="header">
+      <h1>GitHub Tracker</h1>
+    </div>
+  </header>
+
+  <section class="user-data" id="userData"></section>
+
+  <section>
+    <h2>&#128071;&#127997; Technigo Projects &#128071;&#127997;</h2>
```

Comment:
It took a while to understand that the "&#128071;&#127997" was emojis, I was very confused 😅 Maybe in the future make a comment above so it's more clear :)